src/runtime/audience-action-local-flow.ts

```diff
@@ -573,12 +576,47 @@ export class AudienceActionLocalFlow implements AudienceActionFlow {
   private unlock_() {
     removeElement(this.wrapper_);
     setStyle(this.doc_.body, 'overflow', '');
+    this.removeFocusTrap_();
+  }
+
+  private addFocusTrap_() {
+    this.doc_.addEventListener('keydown', this.focusTrap_.bind(this));
+  }
+
+  private removeFocusTrap_() {
+    this.doc_.removeEventListener('keydown', this.focusTrap_.bind(this));
+  }
+
+  private updateFocusable_() {
+    this.focusableElements_ = this.prompt_.querySelectorAll(
+      'a[href], button, input, textarea, select, details, [tabindex]:not([tabindex="-1"])'
+    );
+  }
+
+  private focusTrap_(e: KeyboardEvent) {
+    if (
+      e.key !== 'Tab' ||
```

Comment:
I think we should detect focus events rather than assuming that Tab keydown is the only way focus would be changing. Tab key would not be applicable on mobile for example.

Reply:
Done.